build/rewrite.go

```diff
@@ -151,9 +156,9 @@ func doNotSort(x Expr) bool {
 }
 
 // keepSorted reports whether x is marked with a comment containing
-// "keep sorted", case-insensitive.
+// "keep changed", case-insensitive.
```

Comment:
what's this change?  It does not appear backwards compatible -- I think some internal users make use of keep sorted

Reply:
Reverted.